kalliope/tts/googletts/googletts.py

```diff
@@ -43,35 +43,10 @@ def _generate_audio_file(self):
         .. raises:: FailToLoadSoundFile
         """
 
-        # Prepare payload
-        payload = self.get_payload()
-
-        # getting the audio
-        r = requests.get(TTS_URL, params=payload, stream=True, timeout=TTS_TIMEOUT_SEC)
-        content_type = r.headers['Content-Type']
-
-        logger.debug("Googletts : Trying to get url: %s response code: %s and content-type: %s",
-                     r.url,
-                     r.status_code,
-                     content_type)
-        # Verify the response status code and the response content type
-        if r.status_code != requests.codes.ok or content_type != TTS_CONTENT_TYPE:
-            raise FailToLoadSoundFile("Googletts : Fail while trying to remotely access the audio file")
-
+        # Get rid of insecure request warning
+        requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```

Comment:
Why do you want to get rid of this warning ? 
Also I am wondering if it can be added temporarily, just for this request ? I do not want it to hide warnings from other request coming from kalliope.

Reply:
You are right, this would disable the warning for all next calls. 
Also, we should not see any warnings here as it's a google service and so certainly protected with SSL.

Reply:
The SSL verify is disabled since it was failing in the gTTS package [look here](https://github.com/pndurette/gTTS/issues/36). 
I don't know how to resolve this issue else except of suppress the warning, otherwise we get in each call
```
/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/connectionpool.py:838: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/security.html
  InsecureRequestWarning)
```
In the gTTS package there is already a method to [suppress the warning](https://github.com/pndurette/gTTS/blob/7c502d94c61019dfa993879c64546047d2cc7009/gtts/tts.py#L176-L178), but it seems that this does not effect our call, so I added requests.packages.urllib3.disable_warnings(InsecureRequestWarning) to the code.

Reply:
They disabled the SSL verification from the "request" lib to get it works even behind a proxy.
Could you then reenables warnings after the save? So it doesn't affect the whole kalliope process?

Reply:
Ok thanks for the clarification, all good on my side. Thanks for the PR !